internal/controller/notebook_controller.go

```diff
@@ -424,3 +440,39 @@ func (r *NotebookReconciler) notebookPVC(nb *apiv1.Notebook) (*corev1.Persistent
 
 	return pvc, nil
 }
+
+func (r *NotebookReconciler) callSignedUrlGenerator(notebook *apiv1.Notebook) (string, error) {
+	// TODO(bjb): we should be using TLS here
+	conn, err := grpc.Dial(
+		// TODO(bjb): this doesn't work under `make dev` with port forwarding enabled (telnet returns):
+		// "localhost:10080",
+		// TODO(bjb): transport: Error while dialing dial tcp: lookup gcp-manager: no such host\
+		"gcp-manager:10080",
+		// TODO(bjb): throws: error": "generating upload url: calling the sci service to CreateSignedURL: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp: lookup gcp-manager.substratus.svc.cluster.local: no such host\""
+		// "gcp-manager.substratus.svc.cluster.local:10080",
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
+	)
+	if err != nil {
+		return "", fmt.Errorf("creating an SCI gRPC client: %w", err)
+	}
```

Comment:
Here's where I'm stuck at the moment @samos123 @nstogner

Reply:
unblocked. Unsure what unstuck this (I think the notebook fully reconciling with suspend = false) but I'm able to use localhost when port forwarding and running the controller locally.